src/eko/evolution_operator/__init__.py

```diff
@@ -369,6 +375,7 @@ def quad_ker(self, label, logx, areas):
             logx=logx,
             areas=areas,
             as1=self.a_s[1],
+            as_raw=self.a_s[2],
```

Comment:
Linked to @AleCandido comment: maybe even here putting `as_raw` in between `as0` and `as1` does not make so much sense I believe.

Reply:
it is the same thing ;-) (that Alessandro meant)

Reply:
Exactly ^^